File: csharp/ex052.cs
// Crie um algoritmo que leia a idade de 10 pessoas, mostrando no final:
// a) Qual é a média de idade do grupo
// b) Quantas pessoas tem mais de 18 anos
// c) Quantas pessoas tem menos de 5 anos
// d) Qual foi a maior idade lida


class Ex052
{
  public int counter = 1;
  public int soma = 0;

  public int avgAge = 0;
  public int moreEighteen = 0;
  public int minorFive = 0;

  public int majorAge = 0;

  public void FirstCountdown()
  {
    while (counter <= 10)
    {
      Console.WriteLine($"Digite a idade da pessoa [{counter}]: ");
      int inputAge = Convert.ToInt32(Console.ReadLine());

      soma += inputAge;

      avgAge = soma / 10;

      if (inputAge > 18)
      {
        moreEighteen += 1;
      }
      else if (inputAge < 5)
      {
        minorFive += 1;
      }

      if (inputAge > majorAge)
      {
        majorAge = inputAge;
      }
    }
    Console.Write($"Media de idade do grupo: {avgAge}");
    Console.Write($"Pessoas com mais de 18: {moreEighteen}");
    Console.Write($"Pessoas com menos de 5: {minorFive}");
    Console.Write($"Maior idade do grupo: {majorAge}");
  }
}

File: python/ex052.py
# Crie um algoritmo que leia a idade de 10 pessoas, mostrando no final:
# a) Qual é a média de idade do grupo
# b) Quantas pessoas tem mais de 18 anos
# c) Quantas pessoas tem menos de 5 anos
# d) Qual foi a maior idade lida

counter = 1
avgAge = 0
aboveEighteen = 0
belowFive = 0
majorAge = 0
inputAge = 0
soma = 0

while counter <= 10:
    inputAge = int(input(f"Digite a sua idade pessoa {counter}: "))
    if (inputAge > 18):
        aboveEighteen+=1
    
    if(inputAge < 5):
        belowFive+=1
    
    if(inputAge > majorAge):
        majorAge = inputAge
    
    soma += inputAge
    avgAge = soma/10
    counter+=1
print(f" a) Qual é a média de idade do grupo: {avgAge} \n b) Quantas pessoas tem mais de 18 anos: {aboveEighteen} \n c) Quantas pessoas tem menos de 5 anos: {belowFive} \n d) Qual foi a maior idade lida: {majorAge}")